sqlmesh/core/engine_adapter/base.py

```diff
@@ -1494,6 +1494,10 @@ def _truncate_table(self, table_name: TableName) -> None:
         table = exp.to_table(table_name)
         self.execute(f"TRUNCATE TABLE {table.sql(dialect=self.dialect, identify=True)}")
 
+    @classmethod
+    def _select_columns(cls, columns: t.Iterable[str]) -> exp.Select:
+        return exp.select(*(f'"{c}"' for c in columns))
```

Comment:
Are these identifier quotes? Shouldn't they be dialect specific?

Reply:
no because the context that these are used are sqlglot dialect